klibs.md

```diff
@@ -3,13 +3,14 @@ Klibs
 
 Klibs are plugins which provide extra functionality to unikernels.
 
-As of Nanos [b66be2b](https://github.com/nanovms/nanos/commit/b66be2b642069519a14ea321d9bf5843921c08ab) there are 13 klibs in the kernel source tree:
+As of Nanos [9dc963b](https://github.com/nanovms/nanos/commit/9dc963bb2773b7a3dcb39aade3556350babae16b) there are 19 klibs in the kernel source tree:
 
 * __cloud_init__ - used for Azure && env config init
-  * __cloud_azure__ - use to check in to the Azure meta-data service - not available in config (auto-included by __cloud_init__)
+  * __cloud_azure__ - use to check in to the Azure meta-data service, Azure VM Agent - not available in config (auto-included by __cloud_init__)
```

Comment:
I would just write "implements the Azure VM agent" (checking in to the Azure meta-data service is part of the functionalities of the VM agent)